### ISSUE TITLE
Linked Wiki Pages not visible after install

### ISSUE THREAD

**Issue by ExcaliburVT** [TFSExt/Forms#6]
We just upgraded to TFS2018 SP3 and I installed the Extension.  The linked wiki pages box does not seem to appear and I'm wondering if its because we have some customized templates?  We definitely have the related work box and I assume that you are simply appending to the end of that HTML element?

We definitely have the workitems linked to the Wiki as they show up in the Links tab.

Thanks!

**Comment by boyko-ant** [comment]
Hi!
It seems like a strange behavior. I do not have my own environment with TFS 2018 SP 3, but I can spin it up to check why it happens.
Will keep you posted.

**Comment by ExcaliburVT** [comment]
@boyko-ant have you had any opportunity to look at this, my manager was asking about it and I thought i should follow up.  One other piece of info that might actually be the root cause, we operate in a completely disconnected environment with no internet access.  I didn't see any references to external pages in the source but putting that out there just in case.

Thank you in advance!

**Comment by boyko-ant** [comment]
Hi @ExcaliburVT!

Sorry for such a late answer, I maintain this project at my spare time and I didn't have any in a while. I will do my best to check it on this weekends and provide you with an update. 

Meanwhile, can you please check what you have at javascript console? Maybe there are some errors? It may help if you make a few screenshots and/or copy some error messages here.

**Comment by ExcaliburVT** [comment]
I actually looked there first and did not have any errors. I also looked
specifically for the JavaScript file and it did not seem to be loaded,
which I suspect is the root cause problem. I assume you wrote it against
TFS 2017?  They switched the template system with SP3 and 2018 and I'm
wondering if the JavaScript is simply being left out of the page load. I'm
going to start a new VSTS project and see if I can get it to load there. If
it does it probably points to some thing we have done to our templates you
were not expecting. Thank you for your time invested in this, it will be
really valuable once we figure out the solution.

On Oct 18, 2018 3:41 PM, "Boyko Anton" <notifications@github.com> wrote:

Hi @ExcaliburVT <https://github.com/ExcaliburVT>!

Sorry for such a late answer, I maintain this project at my spare time and
I didn't have any in a while. I will do my best to check it on this
weekends and provide you with an update.

Meanwhile, can you please check what you have at javascript console? Maybe
there are some errors? It may help if you make a few screenshots and/or
copy some error messages here.

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/TFSExt/Forms/issues/6#issuecomment-431135062>, or mute
the thread

**Comment by ExcaliburVT** [comment]
Definitely sounds like it's something on our side. I'm going to get it
running in VSTS so I can see the page load and try to compare that to what
we have on site. Hold off on installing as that is a lot of work you should
not need to do. I will do some more research first.

**Comment by ExcaliburVT** [comment]
@boyko-ant I have definitely been able to see it work on VSTS and researched how Extensions are built and work so I could better diagnose on our server.  Even after creating a brand new TPC I was still unable to get anything to load.  

There are no errors in the browser log and I am able to find the ms.vss-work-web.work-item-form element in the html.  The linked-wiki-pages.html is simply not loaded at all.  

I noted that the extension pulls down the page via a CDN up on VSTS but I suspect that is a VSTS specific thing as I can find no reference to the CDN or external sites anywhere in the vsix (I extracted all the files and went through one by one looking for possible external links etc).  

Our installation is air-gapped and has no connection to the internet at all, but as far as I know that should not cause an issue and I see nothing in the extension source that would require external dependencies.  

In short I'm stumped.  Its like the extension is simply not installing properly even though it says that it is.  I totally understand why you avoid working on any on-prem solution as it results in a lot of issues like this where everyone has their own "special snowflake" installation etc.  Unless someone else comes up and says they have the same issue with an on-prem installation I would not bother exploring it further.  Thank you for all your effort in developing this.

**Comment by boyko-ant** [comment]
Hi @ExcaliburVT!

It may be that the root cause of an issue is not on your side. I played with TFS 2018 and I was unable to make my extension run so smooth as on VSTS (technically I was unable to run it at all). I do have a few more idea how it can be fixed.

For now, as a temporary measure, I narrowed down the installation scope to cloud only.
